src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import java.util.Scanner;
+import duke.task.Task;
+import duke.task.TaskFactory;
+
+public class Duke {
+    private static final String INDENT = "    ";
+    private static final String HORIZONTAL_RULE = "____________________________________________________________\n";
+    private static final String DUKE_LOGO = String.join("\n", " ____        _        ", "|  _ \\ _   _| | _____ ",
+            "| | | | | | | |/ / _ \\", "| |_| | |_| |   <  __/", "|____/ \\__,_|_|\\_\\___|", "");
+    private static final String[] GREETING = { "Hello! I'm Duke", "What can I do for you?" };
+    private static final String EXIT_MESSAGE = "Bye. Hope to see you again soon!";
+
+    private static ListManager listManager = new ListManager();
+
+    public static void displayMessage(String[] lines) {
+        System.out.println(INDENT + HORIZONTAL_RULE);
+        for (String line : lines) {
+            System.out.println(INDENT + " " + line);
+        }
+        System.out.println(INDENT + HORIZONTAL_RULE);
+    }
+
+    public static void displayMessage(String line) {
+        displayMessage(line.split("\n"));
+    }
+
+    public static void processInput(String input) {
```

Comment:
Consider splitting this function into multiple smaller functions since it is a bit cramped with the input processing instructions. It will be more readable that way.